### ISSUE TITLE
[BUG]Popover component has to be updated

### ISSUE THREAD

**Issue by RMCoder198** [italia/design-react-kit#167]
**Describe the bug**
Popover component needs update
**To Reproduce**
1. Go to 'popover'
2. compare it with [Bootstrap italia popover component](https://italia.github.io/bootstrap-italia/docs/componenti/popover/#) 
3. See error

**Expected behavior**
It should be up to date

**Other Required Information (please complete the following information):**
 - OS: [ windows10]
 - Browser [chrome]

**Additional context**
Basically 2 sub-components needs to add Apertura hover and dismissible popover
